wiring/inc/spark_wiring_cloud.h

```diff
@@ -365,7 +366,8 @@ class CloudClass {
 
     static bool connected(void) { return spark_cloud_flag_connected(); }
     static bool disconnected(void) { return !connected(); }
-    static void connect(void) {
+    static void connect(const spark::NetworkClass& network = spark::Network) {
+        spark_set_connection_property(SPARK_CLOUD_BIND_NETWORK_INTERFACE, static_cast<network_interface_t>(network), nullptr, nullptr);
```

Comment:
This should also possibly check current cloud network interface and if they don't match perform a disconnect?

Reply:
I dont see any easy way to disconnect and reconnect if the current cloud connection is on a different network here in the wiring layer. 
`spark_cloud_flag_connect()/spark_cloud_flag_disconnect()` appear to set a flag that is handled in in the system loop. Is there a way to get the system to disconnect immediately from the app/wiring layer?

Reply:
This should probably instead be handled internally in system. E.g. checking if we are connected and bind interface != currently used one and performing disconnect/reconnect.

We could also opt to leave this as-is for now for 5.6.0.

Reply:
Ok, lets handle internally in next release after 5.6.0